test/chi/components/steps.pug

```diff
@@ -0,0 +1,49 @@
+---
+title: Steps
+---
+
+- var steps = ['Create account', 'Choose number', 'Add seats', 'Choose services', 'Checkout'];
```

Comment:
For the test, can we make this more generic and just use "Step 1" "Step 2", etc. for labels